### PR TITLE
Update categories for HOW Phase 0 (bug 1161869)

### PR DESCRIPTION
**Pull request by robhudson**

### CODE REVIEW

File: src/media/js/categories.js [hunk 1]
```diff
@@ -3,23 +3,30 @@ define('categories', ['core/l10n'], function(l10n) {
     var gettext = l10n.gettext;
 
     return [
-        {slug: 'games', name: gettext('Games')},
```
Comment (diox) [inline src/media/js/categories.js]: Don't change the order - it's important, it's the order in which they are displayed

Comment (robhudson) [inline src/media/js/categories.js]: Oh ok. I didn't know that thanks.

Comment (robhudson) [inline src/media/js/categories.js]: It looks like they are alphabetical except for games so I added the new ones in their spot keeping games at the top.

**Comment by diox** [comment]
Should we merge this without waiting for the icons ? Not having the categories is blocking us from running the MOW import script

**Pull request merged**